types/metadataV14.go

```diff
@@ -15,6 +15,7 @@ type MetadataV14 struct {
 	Lookup    PortableRegistry
 	Pallets   []PalletMetadataV14
 	Extrinsic ExtrinsicV14
+	Type      Si1LookupTypeID
```

Comment:
Where do you see this type? I am following the [latest v14 spec](https://github.com/polkadot-js/api/blob/48ef04b8ca21dc4bd06442775d9b7585c75d1253/packages/types/src/interfaces/metadata/v14.ts#L30-L34) and it's not there 🕵️

Reply:
check master branch [master](https://github.com/polkadot-js/api/blob/0d0a230c860dfd9c2079dc341b0fa61387ad4b1d/packages/types/src/interfaces/metadata/v14.ts#L36-L40)

Reply:
Thank you!